manifests/plugin/remote_execution/cockpit.pp

```diff
@@ -25,41 +32,56 @@
   }
   $cockpit_origins = [$foreman_url] + $origins
 
-  foreman::plugin { 'remote_execution-cockpit': }
-  -> service { 'foreman-cockpit':
-    ensure => running,
-    enable => true,
+  foreman::plugin { 'remote_execution-cockpit':
+    version => $ensure,
   }
 
+  if $ensure != 'absent' {
+    service { 'foreman-cockpit':
```

Comment:
Why doesn't this leave the service running?

Reply:
I didn't verify it, but I'm counting on packaging to drop it. I think this should cover it:
https://github.com/theforeman/foreman-packaging/blob/6656cf0adbbd5097ed16fa039301bcb56090a42f/packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec#L117-L118